### ISSUE TITLE
Is Task support via DAV possible yet?

### ISSUE THREAD

**Issue by feedhopper** [davish/obsidian-full-calendar#173]
Title says it all. I very much look forward to using this plugin as I find the tasks integration lacking in too many places, especially when it comes to other programs that already store my calendar and task events as DAV.

So: Is this already possible?
If yes: HOW?
:)))
If not: is it planned?
If yes: How high is it on the priority list?

THANKS for this VERYPROMISING implementation.

**Comment by davish** [comment]
duplicate of #45 and #25.

CalDAV sync is a tough nut to crack, task support is one of the next things up after #103.

**Comment by feedhopper** [comment]
I am sorry about this. I did search for task related issues but it is a little difficult to navigate the many requests. Glad to see its next up on the list.

and ... THANK YOU!
